src/main/kotlin/com/odoo/odools/OdooLSLspServerSupportProvider.kt

```diff
@@ -16,12 +16,12 @@ val osName = System.getProperty("os.name").lowercase()
 val arch = System.getProperty("os.arch").lowercase()
 
 val targetOs = when {
-    osName.contains("win") && arch.contains("64") -> "win32-x64"
-    osName.contains("win") && arch.contains("arm") -> "win32-arm64"
-    osName.contains("linux") && arch.contains("64") -> "linux-x64"
-    osName.contains("linux") && arch.contains("arm") -> "linux-arm64"
-    osName.contains("darwin") && arch.contains("64") -> "darwin-x64"
-    osName.contains("darwin") && arch.contains("arm") -> "darwin-arm64"
+    SystemInfo.isWindows && !SystemInfo.isAarch64 -> "win32-x64"
+    SystemInfo.isWindows && SystemInfo.isAarch64 -> "win32-arm64"
```

Comment:
The check for x86 64 bit and AM 64 are both the same. `isAarch64`.

Shouldn't they be different? 

Same for Linux and MacOS